sdk/contentsafety/azure-ai-contentsafety/samples/sample_manage_blocklist.py

```diff
@@ -0,0 +1,189 @@
+import os
+from azure.ai.contentsafety import ContentSafetyClient
+from azure.core.credentials import AzureKeyCredential
+from azure.ai.contentsafety.models import TextBlocklist, TextBlockItemInfo, AddBlockItemsOptions, RemoveBlockItemsOptions, AnalyzeTextOptions
+from azure.core.exceptions import HttpResponseError
+import time
+
+
+key = os.environ["CONTENT_SAFETY_KEY"]
+endpoint = os.environ["CONTENT_SAFETY_ENDPOINT"]
+
+# Create an Content Safety client
+client = ContentSafetyClient(endpoint, AzureKeyCredential(key))
+
+def list_text_blocklists():
+    try:
+        return client.list_text_blocklists()
+    except HttpResponseError as e:
```

Comment:
we shouldn't include all of this error handling in the sample. Instead, we should loop oer the test blocklists and print out the main attributes on them, i.e.
```python
for text in client.list_test_blocklists():
    print(text.blocklist_name)
```

Reply:
same comment for the other samples in this file: focus less on the error handling and showcasing the important parts of the response

Reply:
Added code to parse attributes returned.